fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -145,6 +157,49 @@ def load_data(self, X, y, **kwargs):
         """Load the specified data into the object."""
         super().load_data(X, y, event=_ALL, **kwargs)
 
+class EqualOpportunity(ConditionalSelectionRate):
+    r"""Implementation of Equalized Odds as a moment.
+
+    Adds conditioning on label compared to Demographic parity, i.e.
+
+    .. math::
+       P[h(X) = y' | A = a, Y = y] = P[h(X) = y' | Y = y] \; \forall a, y, y'
+
+    This implementation of :class:`ConditionalSelectionRate` defines
+    events corresponding to the unique values of the `Y` array.
+
+    The `prob_event` :class:`pandas:pandas.DataFrame` will record the
+    fraction of the samples corresponding to each unique value in
+    the `Y` array.
+
+    The `index` MultiIndex will have a number of entries equal to
+    the number of unique values for the sensitive feature, multiplied by
+    the number of unique values of the `Y` array, multiplied by two (for
+    the Lagrange multipliers for positive and negative constraints).
+
+    With these definitions, the :meth:`signed_weights` method
+    will calculate the costs according to Example 4 of
+    `Agarwal et al. (2018) <https://arxiv.org/abs/1803.02453>`_.
+    """
+
+    short_name = "EqualizedOdds"
```

Comment:
please update the short_name as well

Reply:
unless this is indeed EO. The class name threw me off

Reply:
Oops yeah, good catch sorry about this. I think I messed up a merge before raising this PR because of which it got added twice!

Reply:
No worries!